### DISCUSSION TITLE
How do you keep track of pnpm version?

### DISCUSSION THREAD

**Discussion by Bielik20** [pnpm/pnpm#8271] [Q&A] [answered]
Where I work I am responsible for setup of other developer tools.

What I use right now is `.nvmrc` with zsh config that automatically uses appropriate Node version. I do it because with `.nvmrc` both CI and local setups use the same Node and **npm version** (we just use default npm version for installed Node version).

So my question is about the latter part. How do I make sure that everyone on a team uses the same `pnpm` version? I would like that to be as simple and automated as possible. I just want to underline here: I do not want developers to have to check and pick pnpm version themselves (people will inevitably mess that up 😅).

Any help is appreciated.

**Comment by zkochan** [comment, marked as answer]
If you want everyone to use the same version, you can use corepack: https://pnpm.io/installation#using-corepack

**Reply by Bielik20** [reply]
It works really great and integrates well with GitHub Actions, thanks 🙇
